analysis_options.yaml

```diff
@@ -129,7 +129,7 @@ linter:
     # - only_throw_errors # https://github.com/flutter/flutter/issues/5792
     - overridden_fields
     - package_api_docs
-    - package_names
+    # - package_names
```

Comment:
Could you add a comment to each of these so we remember why they are disabled? something like

package_names: non conforming packages in sdk
sort_pub_dependencies: prevents separating pinned transitive dependencies

Reply:
Good idea.  Done!